docs/guides/admin/docs/workflowoperationhandlers/publication-to-workspace-woh.md

```diff
@@ -0,0 +1,36 @@
+# PublicationToWorkspaceWorkflowOperationHandler
+
+## Description
+The CopyWorkflowOperationHandler can be used to copy the content form publication channels to workspace.
+Wit this Workflow one can copy or maipulate published elements without reencoding.
+## Parameter Table
+
+|Configuration Key  |Example           |Description                                           |
+|-------------------|------------------|------------------------------------------------------|
+|publication-channel|engage-player     |the publication-channel interal,engage-player,oaipm   |
```

Comment:
Based on the description, I'm not sure if this is supposed to be one value or a comma-separated list.